$OpenBSD: patch-taglib_ape_apeproperties_cpp,v 1.3 2012/03/20 16:23:47 dcoppa Exp $

Fix: crash parsing ID3v2 tag in ape file due to null pointer
dereference
(upstream git commit 7cc36db7606dfc85d2e344d35c4e26fe8f698bdc)

--- taglib/ape/apeproperties.cpp.orig	Sun Mar 18 16:00:45 2012
+++ taglib/ape/apeproperties.cpp	Sun Mar 18 16:01:58 2012
@@ -137,7 +137,7 @@ long APE::Properties::findDescriptor()
   long ID3v2OriginalSize = 0;
   bool hasID3v2 = false;
   if(ID3v2Location >= 0) {
-    ID3v2::Tag tag(d->file, ID3v2Location, 0);
+    ID3v2::Tag tag(d->file, ID3v2Location);
     ID3v2OriginalSize = tag.header()->completeTagSize();
     if(tag.header()->tagSize() > 0)
       hasID3v2 = true;
